src/bioseq/site_counting/site_counting.jl

```diff
@@ -89,20 +89,20 @@ for A in (DNAAlphabet, RNAAlphabet)
 
         # Match
         @inline function bp_chunk_count(::Type{Match}, ::Type{$A{4}}, a::UInt64, b::UInt64)
-            return count_zero_nibbles(a $ b)
+            return count_zero_nibbles(a ⊻ b)
         end
 
         @inline function bp_chunk_count(::Type{Match}, ::Type{$A{2}}, a::UInt64, b::UInt64)
-            return count_zero_bitpairs(a $ b)
+            return count_zero_bitpairs(a ⊻ b)
```

Comment:
Is this necessary? Is `$` being changed? It's easy to type the `$` character, not so easy to type \veebar. (The only reason I knew it was \veebar is because I know LaTeX! Others will not know).